lib/Logger.jsx

```diff
@@ -81,60 +73,51 @@ export default class Logger {
      * @param {String} message The message to log.
      * @param {Boolean} sendNow if true, the message will be sent right away.
      * @param {Object|String} parameters The parameters to send along with the message
+     * @param {Boolean} onlyFlushWithOthers A request will never be sent to the server if all loglines have this set to true
      */
-    info(message, sendNow, parameters) {
-        if (sendNow) {
-            const msg = `[info] ${message}`;
-            this.logToServer(msg, 0, parameters);
-        } else {
-            this.add(message, parameters);
-        }
+    info(message, sendNow = false, parameters= '', onlyFlushWithOthers = false) {
+        const msg = `[info] ${message}`;
+        this.add(msg, parameters, sendNow, onlyFlushWithOthers);
     }
 
     /**
      * Logs an alert.
      *
      * @param {String} message The message to alert.
-     * @param {Number} recentMessages A number of recent messages to append as context
      * @param {Object|String} parameters The parameters to send along with the message
      * @param {Boolean} includeStackTrace Must be disabled for testing
      */
-    alert(message, recentMessages, parameters = {}, includeStackTrace = true) {
+    alert(message, parameters = {}, includeStackTrace = true) {
```

Comment:
I think we are using this Logger in `Web-Expensify` (possibly elsewhere) and should make sure the method signature change doesn't screw things there?

Reply:
Oh! nvm I see there are other PRs already :ohnothing: